hdf_compass/adios_model/model.py

```diff
@@ -19,6 +19,7 @@
 import sys
 import os.path as op
 import posixpath as pp
+import six
```

Comment:
The idea is to switch to Python 3 and avoid to increase the code complexity with all the `if`'s. Look at the other code in the `py3` branch.

Reply:
So you want Python 3+? Fine with me, sure